spec/models/article_spec.rb

```diff
@@ -5,4 +5,7 @@
     it { should validate_presence_of(:title) }
     it { should validate_presence_of(:content) }
   end
+  describe "associations" do
```

Comment:
**nitpick:** Add a new line.

Everybody loves a bit of breathing space. Let your code breathe!